metrics/report/report_dockerfile/metrics_report.Rmd

```diff
@@ -87,19 +87,6 @@ source('fio-writes.R')
 ```
 \pagebreak
 
-# Network CPU costs
```

Comment:
OOI why was this removed?

Reply:
@Jakob-Naucke currently we do not have this script running on the metrics CI